src/test/java/com/dougnoel/sentinel/filemanagers/DownloadManagerTests.java

```diff
@@ -18,5 +18,25 @@ public void testVerifyTextNotInDownloadedPDFPageTwo() {
 		File file = new File("src/test/resources/pdfs/TestPDF.pdf");
 		assertFalse("Expecting text to not appear on page 2 of PDF.", DownloadManager.verifyTextInDownloadedPDF("This is page one.", file, 2, 2));
 	}
-
+	
+	@Test
+	public void testVerifyPDFComparisonViaRendering() {
+		File file = new File("src/test/resources/pdfs/TestPDF.pdf");
+		File file2 = new File("src/test/resources/pdfs/TestPDF_copy.pdf");
+		assertTrue("Expecting PDF to match a copy of itself.", DownloadManager.verifyDownloadedPDFViaVisualRendering(file, file2));
+	}
+	
+	@Test
+	public void testVerifyPDFComparisonViaRendering_Negative() {
+		File file = new File("src/test/resources/pdfs/TestPDF.pdf");
+		File file2 = new File("src/test/resources/pdfs/TestPDF_different.pdf");
+		assertFalse("Expecting PDF to match a copy of itself.", DownloadManager.verifyDownloadedPDFViaVisualRendering(file, file2));
+	}
+	
+	@Test
+	public void testVerifyPDFComparisonViaRendering_OutputFileError() {
+		File file = new File("src/test/resources/pdfs/TestPDF.pdf");
+		File file2 = new File("src/test/resources/pdfs/TestPDF_different.pdf");
+		assertFalse("Expecting PDF to match a copy of itself.", DownloadManager.verifyDownloadedPDFViaVisualRendering(file, file2, 0.0, "/bin/bash"));
```

Comment:
Pretty sure this message is not correct but I don't understand why this is different.